sdk/python/tests/fixtures/data_source_fixtures.py

```diff
@@ -63,3 +67,13 @@ def prep_bq_source(df, event_timestamp_column="") -> BigQuerySource:
     return BigQuerySource(
         table_ref=table_ref, event_timestamp_column=event_timestamp_column,
     )
+
+
+def simple_bq_source_using_query_arg(df, event_timestamp_column="") -> BigQuerySource:
+    bq_source_using_table_ref = simple_bq_source_using_table_ref_arg(
+        df, event_timestamp_column
+    )
+    return BigQuerySource(
+        query=bq_source_using_table_ref.table_ref,
```

Comment:
The query is supposed to be a select statement, so I think something like f"SELECT * FROM {bq_source_using_table_ref.table_ref}" would be better